js/application.js

```diff
@@ -1,30 +1,39 @@
 $(document).ready(function(){
-	
-	var postworkshop = $.getJSON('json/postworkshop.json', function(data){
-		var reviews = $.map(data, function(row){
+	var scheduleData, reviews;
```

Comment:
You might want to save the promises rather than the data from the promises.  This would make the `when()` a little cleaner and allow you to do more with the promises if you had to.